script/universal/MultisigBase.sol

```diff
@@ -9,6 +9,7 @@ import "./Simulator.sol";
 
 abstract contract MultisigBase is Simulator {
     IMulticall3 internal constant multicall = IMulticall3(MULTICALL3_ADDRESS);
+    bytes32 internal constant SAFE_NONCE_SLOT = bytes32(uint256(5));
```

Comment:
There is risk this will collide with another value that is automatically assigned by solc